assets/src/edit-story/elements/video/output.js

```diff
@@ -34,7 +34,10 @@ function VideoOutput({ element, box }) {
 
   const props = {
     autoPlay: 'autoplay',
-    poster: defaultForUndefined(element.poster, resource.poster),
+    poster: defaultForUndefined(
+      defaultForUndefined(element.poster, resource.poster),
+      ''
+    ),
```

Comment:
This looks horrible!

Could we change the function to just take `...args` and pick the first non-`undefined`  - and thus do `defaultForUndefined(element.poster, resource.poster, '')` here? Maybe even rename the function then?

Reply:
Sure. Might be  worth moving that to utils too so it could be used in places like `getCommonValue()`

Reply:
For comparison, Lodash uses [`defaultTo`](https://lodash.com/docs/4.17.15#defaultTo) for such a util.